docs/README.md

```diff
@@ -1,11 +1,20 @@
 # ideacrm
 
-VTEX Idea CRM first-party app
+The CRM Platform's omnichannel enables complete control of customer interactions with the enterprise, providing measurements and traceability. Its service channels are Chat, Phone, Email, SMS, Tickets, Social Networks and Applications.
+For more information please visit, [idecrm](https://www.gvp.com.br/crm) website.
 
 ## How to install
 
 Open the VTEX App Store and install this app on your store.
```

Comment:
1. Open the **VTEX App Store** and install the Idea CRM app on your store. You can also run the following command in your terminal: 

```sh
vtex install vtex.ideacrm@0.x
```
2. In the admin's side bar, access **Apps**  within the **Account settings** section. 
3. Select the **Idea CRM** box and fill out the necessary fields in order to complete the app's configuration.

Reply:
I only added 3 steps because thats the info available in the docs. If there are any more required steps the user should know about, I believe it should be stated here as well... such as all mandatory fields and how it should be fill out by the user